public/try.html

```diff
@@ -1 +0,0 @@
-../try.html
```

Comment:
Could you reset these files? ^^

Reply:
Strange, did not see that when i merged.
Will try get them back!